### PR TITLE
feat(monitor): update docs

### LINKED ISSUE #1368: need configuration change guide when using customized configuration file in tidb-monitor
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.
in doc https://docs.pingcap.com/tidb-in-kubernetes/stable/monitor-a-tidb-cluster#use-a-customized-configuration-file, it doesn't show how to change the configuration after deploy tidb-monitor using customized configuration file.

2. Describe your suggestion or addition.
In my test, it can be done with 2 steps:
edit the ConfigMap
delete & apply to recreate the tidb-monitor

there are 3 points need emphasize
spec.persistent 
spec.pvReclaimPolicy 
external service port will be changed

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by just1900**
Signed-off-by: just1900 <legendj228@gmail.com>

### What is changed, added, or deleted? (Required)

close #1368 

1. adding prometheusReloader to all examples in docs

2. refine docs for using customized configurations

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Review by csuzhangxc — APPROVED** [review]
那个 `v0.49.0` 的版本选择有什么讲究

**Comment by just1900** [comment]
> 那个 `v0.49.0` 的版本选择有什么讲究

0.0 没什么讲究，就是和之前 examples 里已有的 version 保持一样。
这个镜像版本是跟着 prometheus-operator 走的，估计不同版本也没什么功能上的变动

File: en/monitor-a-tidb-cluster.md [hunk 1]
```diff
@@ -77,6 +82,7 @@ You can customize the Prometheus configuration by using a customized configurati
 
 1. Create a ConfigMap for your customized configuration, and set the key name of `data` to `prometheus-config`.
 2. Set `spec.prometheus.config.configMapRef.name` and `spec.prometheus.config.configMapRef.namespace` to the name and namespace of the customized ConfigMap respectively.
+3. Check if TidbMonitor has enabled [dynamic-configuration](enable-monitor-dynamic-configuration.md), if not, you need to restart TidbMonitor's pod to reload the configuration.
```
Comment (ran-huang) [inline en/monitor-a-tidb-cluster.md:85]:
```suggestion
3. Check if TidbMonitor has enabled [dynamic configuration](enable-monitor-dynamic-configuration.md). If not, you need to restart TidbMonitor's pod to reload the configuration.
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**